fixtures/allauth.json.tmpl

```diff
@@ -14,19 +14,5 @@
             "domain":"127.0.0.1:8000",
             "name":"127.0.0.1:8000"
         }
-    },
```

Comment:
Okay, so where is the description in the PR on how to make this change happen? And remove this from the DB?

Reply:
I have added that